src/app/app.component.html

```diff
@@ -1,49 +1,126 @@
-<div class="wrapper  w-full flex justify-center">
+<div class="wrapper w-full flex justify-center">
   <div class="w-full max-w-[1440px]">
-    <nav class="bg-transparent w-full z-20 text-black flex justify-between items-center h-[90px] max-w-[1200px] mx-auto mb-[57px]">
-      <div class="h-[70px] justify-between items-center w-full flex">
+    <nav
+      class="w-full bg-[#FFFFFF] fixed md:static z-20 text-black flex justify-between items-center h-[90px] max-w-[1200px] mx-auto lg:mb-[57px] md:mb-0 px-5 md:px-0">
+      <div class="h-[60px] md:h-[70px] justify-between items-center w-full flex">
         <div class="flex items-center">
-          <a href="#">
-            <img src="assets/new_logo.svg" alt="Bee O`clock Logo">
+          <a href="#" class="flex gap-[14px]">
+            <img ngSrc="assets/new_logo.svg" alt="Bee O`clock Logo" height="38" width="38">
+            <img class="hidden md:block" ngSrc="assets/logo_text.svg" alt="Bee O`clock" height="16" width="119">
           </a>
         </div>
+        <div>
+          <img class="md:hidden block" ngSrc="assets/logo_text.svg" alt="Bee O`clock" height="16" width="119">
+        </div>
+        <div class="md:hidden">
+          <button (click)="toggleMobileMenu()" class="text-black focus:outline-none h-[100%]">
+            <ng-icon [class]="isMobileMenuOpen ? 'bi bi-x' : 'bi bi-three-dots'"></ng-icon>
+          </button>
+        </div>
         <ul class="hidden md:flex gap-[60px]">
           <li><a href="#" class="hover:text-yellow-500 font-[FixelDisplay-Medium] text-[15px]">Послуги</a></li>
           <li><a href="#" class="hover:text-yellow-500 font-[FixelDisplay-Medium] text-[15px]">Тарифи</a></li>
           <li><a href="#" class="hover:text-yellow-500 font-[FixelDisplay-Medium] text-[15px]">Відгуки</a></li>
           <li><a href="#" class="hover:text-yellow-500 font-[FixelDisplay-Medium] text-[15px]">FAQ</a></li>
           <li><a href="#" class="hover:text-yellow-500 font-[FixelDisplay-Medium] text-[15px]">Про нас</a></li>
         </ul>
-
-        <div class="flex items-center space-x-4">
+        <div class="items-center space-x-4 hidden md:flex">
           <a href="#" class="hover:text-yellow-500 font-[FixelDisplay-Medium] text-[15px]">Вхід</a>
-          <utility-change-language-component></utility-change-language-component>
         </div>
+        @if (isMobileMenuOpen) {
+          <div class="fixed inset-0 bg-white z-30 md:hidden mt-[70px] overflow-auto">
+            <ul class="flex flex-col items-center px-5 pt-4 [&>li]:border-b [&>li]:border-[#F4F4F5]">
+              <li class="flex items-center h-[51px] w-full">
```

Comment:
Can you prepare in `.ts` file array variable and use the new array variable to build the `li` list in html via `@for` ?

TIP:
And you also can move the classes `flex items-center h-[51px] w-full` into `ul` as you did with `[&>li]:border-b`

Reply:
done